src/internal/testutil/cmds.go

```diff
@@ -133,51 +136,37 @@ func BashCmd(cmd string, subs ...string) *exec.Cmd {
 	res.Env = os.Environ()
 	return res
 }
-func PachctlBashCmd(t *testing.T, c *client.APIClient, cmd string, subs ...string) *exec.Cmd {
-	t.Helper()
 
-	buf := new(bytes.Buffer)
-	_, err := buf.ReadFrom(bashCmd(cmd, subs...))
-	require.NoError(t, err)
+func subsToTemplateData(subs ...string) (map[string]string, error) {
```

Comment:
Maybe we could have `bashCmd()` call this function instead of repeating the same steps.